app/core/downloader/episode.py

```
@@ -94,4 +104,13 @@ def build_episode(
                     getattr(ep, "episode", episode),
                     err,
                 )
+    if site == "s.to":
+        try:
+            from app.providers.sto.v2 import enrich_episode_from_v2_url
+
+            if isinstance(base_url, str) and base_url:
+                enrich_episode_from_v2_url(episode=ep, base_url=base_url)
+        except Exception as err:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)